src/components/GenericPlaceholder.tsx

```diff
@@ -22,6 +22,17 @@ interface GenericPlaceholderNoButtonProps
   buttonVariant?: never
 }
 
+interface GenericPlaceholderNoButtonAndTitleProps
```

Comment:
Well... No ^^
Just change `GenericPlaceholderProps` and make `title` optional. No need to have a full other type for that.

Reply:
Fine with that ! 😄 Will do

Reply:
Done